SPECS/kernel/config_aarch64

```diff
@@ -1045,11 +1045,11 @@ CONFIG_ZSWAP_COMPRESSOR_DEFAULT_LZO=y
 # CONFIG_ZSWAP_COMPRESSOR_DEFAULT_ZSTD is not set
 CONFIG_ZSWAP_COMPRESSOR_DEFAULT="lzo"
 CONFIG_ZSWAP_ZPOOL_DEFAULT_ZBUD=y
-# CONFIG_ZSWAP_ZPOOL_DEFAULT_Z3FOLD is not set
+# CONFIG_ZSWAP_ZPOOL_DEFAULT_Z3FOLD_DEPRECATED is not set
 # CONFIG_ZSWAP_ZPOOL_DEFAULT_ZSMALLOC is not set
 CONFIG_ZSWAP_ZPOOL_DEFAULT="zbud"
 CONFIG_ZBUD=y
-CONFIG_Z3FOLD=m
+# CONFIG_Z3FOLD_DEPRECATED is not set
```

Comment:
z3fold is being deprecated as it has latent bugs and the alternatives of zsmalloc and zbud offer comparable if not better memory usage and performance. The full commit is shown here: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/54ad9c7608543460ecead776bff7ebbba993b73a

Reply:
Agreed - Looks to be no impact to our users since we're set to `zbud` by default.